readme.md

```diff
@@ -135,6 +135,8 @@ The debug will show you an array like this:
 
 ### Differences
 
+
+* This plugin does intentionally **not** handle authorization. It was [decoupled](https://en.wikipedia.org/wiki/Coupling_(computer_programming)) from authorization on purpose. 
```

Comment:
Is it worth mentioning that we might have another plugin for doing authorization?